services/ui-src/src/components/reports/ReportPageFooter.tsx

```diff
@@ -20,56 +20,57 @@ export const ReportPageFooter = ({
     report?.formTemplate.flatRoutes,
     report?.formTemplate.basePath
   );
-  const hidePrevious = previousRoute === "/mcpar" || previousRoute === "/mlr";
 
-  const { userIsAdmin, userIsReadOnly } = useStore().user ?? {};
+  const { userIsAdmin, userIsReadOnly, userIsEndUser } = useStore().user ?? {};
   const isAdminUserType = userIsAdmin || userIsReadOnly;
-  const formIsDisabled = isAdminUserType && !form?.editableByAdmins;
+  const hidePrevious = previousRoute === "/mcpar" || previousRoute === "/mlr";
+  const reportWithSubmittedStatus = report?.status === ReportStatus.SUBMITTED;
+  const formIsDisabled =
+    (isAdminUserType && !form?.editableByAdmins) ||
+    (userIsEndUser && reportWithSubmittedStatus);
+  const isReadOnly = !form?.id || formIsDisabled;
+
+  const prevButton = (
+    <Button
+      onClick={() => navigate(previousRoute)}
+      variant="outline"
+      leftIcon={<Image src={previousIcon} alt="" sx={sx.arrowIcon} />}
```

Comment:
we lost the alt text on the icons for this button and the rest

Reply:
Good catch